python/pyspark/context.py

```diff
@@ -244,26 +269,32 @@ def _do_init(
         self.environment["PYTHONHASHSEED"] = os.environ.get("PYTHONHASHSEED", "0")
 
         # Create the Java SparkContext through Py4J
-        self._jsc = jsc or self._initialize_context(self._conf._jconf)
+        self._jsc = jsc or self._initialize_context(self._conf._jconf)  # type: ignore[attr-defined]
```

Comment:
We can add this to the corresponding stub file, so we can skip ignore, but I won't insist on that.